safe_eth/eth/account_abstraction/bundler_client.py

```diff
@@ -173,13 +179,20 @@ def get_user_operation_and_receipt(
             self._get_user_operation_receipt_payload(user_operation_hash, request_id=2),
         ]
         result = self._do_request(payload)
-        if not (result and result[0]):
+        if (
```

Comment:
Why was necessary invert the condition?

Reply:
Just for simplicity, as the `_do_request` function returns different types, mypy forces you to check the type of each element that is accessed afterwards. This way I feel it is clearer.

Reply:
Makes sense